hr_timesheet_overtime/models/account_analytic_line.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+from openerp import models, fields, api
+
+_logger = logging.getLogger(__name__)
+
+
+class AnalyticLine(models.Model):
+    """
+    Apply on account analytic lines the rate defined in resource.overtime.rate
+    """
+
+    _inherit = "account.analytic.line"
+
+    @api.model
+    def create(self, values):
+        self._update_values(values)
+        return super(AnalyticLine, self).create(values)
+
+    @api.multi
+    def write(self, values):
+        self._update_values(values)
+        return super(AnalyticLine, self).write(values)
+
+    @api.model
+    def _update_values(self, values):
+        if values.get("date") or values.get("unit_amount"):
+            date = values.get("date")
```

Comment:
if you have `unit_amout` in vals and no `date`, `date` will be None here.

Reply:
Indeed, Cath got the error while testing :boom:. Fixed !